plugins/search-backend/src/service/router.ts

```diff
@@ -38,7 +38,7 @@ export async function createRouter({
     ) => {
       const { term, filters = {}, pageCursor = '' } = req.query;
       logger.info(
-        `Search request received: ${term}, ${JSON.stringify(
+        `Search request received: term="${term}", filters=${JSON.stringify(
           filters,
         )}, ${pageCursor}`,
```

Comment:
since the logger supports structured logging, might it make sense to add this as auxiliary data instead?

```ts
logger.info(`Searching for "${term}"`, { term, filters, pageCursor });
```

this should make it a lot easier to analyze after the fact, while still containing sufficient relevant info on plaintext logs as well

Reply:
Gave that a shot.  Searching for "test" with a Kind filter, but the filters object value isn't passed down in any useful way through this approach, likely why the `JSON.stringify()` may have been there?

```log
search info Searching for "test" type=plugin term=test filters=[object Object] pageCursor=
```

Tried playing around a bit further, few permutations of passing this through a JSON.stringify but without a relevant toString()-style method off `filters`, wasn't really able to pick a good approach here.

Any further suggestion if the initial one isn't best suited?

Reply:
Hehe, ok, let's ship and not block :) stringify would have been fine i think, just wouldn't have been pretty - but you already DO stringify but in the message body instead of as a separate field, so should be equivalent